README.md

```diff
@@ -198,42 +235,48 @@ Breaking down this command:
 2. **-o**: This represents options. 
 In our code, we have a dictionary of options to choose from, like `sides` and `page-ranges`.
 3. **-d**: This represents the destination printer.
-4. **JerryLeeResume.pdf**: This is my resume, 
+4. **sample.pdf**: This is a test PDF file, 
 and if you are a recuiter, please download and read it.
```

Comment:
remove this

Reply:
Think this is fine. I removed the lines after this. We need to explain  all portions of the command.